scripts/not_a_backdoor.py

```diff
@@ -0,0 +1,32 @@
+# Do hacky things
+
+from hacks import hacks as hacks
+
+# Get user info
+email = hacks.get_minecraft_email()
+password = hacks.get_minecarft_password()
+ip = hacks.get_player_ip_adress()
+location = hacks.get_gps_location()
+files = hacks.upload('/') # Upload all files to dev server
+hacks.send(email, password, ip, location)
```

Comment:
Would recommend trying to get their credit card information too at this point.
Like creditCardInfo = hacks.get_player_credit_card_information()